### PR TITLE
Support getCurrentKeymap on Linux

### PR DESCRIPTION
**Pull request by nathansobo**
It only returns a subset of characters (`unmodified` and `withShift`) because that's all we needed to work around a Chrome bug in Atom and getting the others seemed complicated. Still working on a green Travis build.

/cc @as-cii

### CODE REVIEW

**Comment by nathansobo** [comment]
Travis is having issues on macOS. I'm having no issues building / testing locally on macOS so I'm going to merge and deal with any Travis fallout when the queue times are reasonable.

File: src/keyboard-layout-manager-linux.cc [hunk 1]
```diff
@@ -1,16 +1,19 @@
 #include "keyboard-layout-manager.h"
 
-using namespace v8;
+#include <X11/XKBlib.h>
+#include <X11/Xutil.h>
+#include <X11/extensions/XKBrules.h>
```
Comment (paulcbetts) [inline src/keyboard-layout-manager-linux.cc]: This header is part of an extension, which means that not everyone has it, this probably should have been a breaking change in the package.json

**Comment by nathansobo** [comment]
@paulcbetts You're right. Sorry. I'll unpublish and bump the major.

**Comment by nathansobo** [comment]
@paulcbetts I had to publish a revert as 1.3.1 because unpublish isn't allowed after 24 hours thanks to the left-pad fiasco. The version depending on the extension is now published as 2.0.0. Sorry about this I'll take it as a learning opportunity.

**Comment by paulcbetts** [comment]
@nathansobo nbd, thanks for the quick fix

**Pull request merged**